include/f_core/flight/CPhaseController.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+#include <array>
+#include <cstdint>
+#include <cstdio>
+#include <zephyr/kernel.h>
+
+void flight_log_init();
```

Comment:
RN these are just stubbed out logging but for an actual flight we would have these open a file and write events to that file as the flight happens so that we can get a better idea of what the computer was thinking rather than having to do forensics to get that after the fact